dbsync.py

```diff
@@ -671,9 +679,13 @@ def init(conn_cfg, mc, from_gpkg=True):
                 print("The GPKG file, base and modified schemas are already initialized and in sync")
                 return  # nothing to do
         elif modified_schema_exists:
-            raise DbSyncError(f"The 'modified' schema exists but the base schema is missing: {conn_cfg.base}")
+            raise DbSyncError(f"The 'modified' schema exists but the base schema is missing: {conn_cfg.base}. "
+                              f"Schema `{conn_cfg.modified}` should be removed or renamed in the database "
+                              "so that DB Sync can set up sync properly.")
```

Comment:
```suggestion
                              f"This may be a result of a previously failed attempt to initialize DB sync. You can delete schema `{conn_cfg.modified}` in the database to fix this error and restart DB sync.")
```